e2e/cypress/integration/enterprise/group_mentions/group_mentions_permissions_spec.js

```diff
@@ -113,8 +113,8 @@ describe('Group Mentions', () => {
         // # Disable Group Mentions for All Users & Channel Admins
         cy.visit('/admin_console/user_management/permissions/system_scheme');
         cy.get('.admin-console__header', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible').and('have.text', 'System Scheme');
-        disablePermission('all_users-posts-use_group_mentions-checkbox');
-        disablePermission('channel_admin-posts-use_group_mentions-checkbox');
+        disablePermission('all_users-integrations-use_group_mentions-checkbox');
```

Comment:
This is the bug https://mattermost.atlassian.net/browse/MM-42020
Permission checkboxes have been moved to wrong category
Same for all other permission changes in this PR